### PR TITLE
DEVEX-683 dx describe a global workflow

### PR DESCRIPTION
**Pull request by commandlinegirl**

### CODE REVIEW

**Comment by commandlinegirl** [comment]
I added an example `$dx describe` output to the JIRA ticket. HTH :-)

File: src/python/dxpy/scripts/dx.py [hunk 1]
```diff
@@ -1080,6 +1080,62 @@ def tree(args):
         err_exit()
 
 def describe(args):
+
+    def describe_global_executable(json_output, args, exec_type):
+        """
+        Describes a global executable, i.e.e either app or global workflow
```
Comment (geetduggal) [inline src/python/dxpy/scripts/dx.py]: Extra 'e' here.

File: src/python/dxpy/scripts/dx.py [hunk 2]
```diff
@@ -1080,6 +1080,62 @@ def tree(args):
...
+                print_desc(desc, args.verbose)
+            found_match = True
+        except dxpy.DXAPIError as details:
+            if details.code != requests.codes.not_found:
+                raise
+        return found_match
+
+    def find_global_executable(json_output, args):
+        """
+        Makes a find_apps API call and, if no matches are found, find_global_workflows call.
```
Comment (geetduggal) [inline src/python/dxpy/scripts/dx.py]: 'are found, *a* find_global_workflows call'

File: src/python/dxpy/utils/describe.py [hunk 3]
```diff
@@ -566,19 +571,76 @@ def print_app_desc(desc, verbose=False):
...
+                    print_field("Workflow ID", workflow_desc['id'])
+                if workflow_desc.get('inputSpec') is not None and workflow_desc.get('inputs') is None:
+                    print_nofill_field("Input Spec", get_io_spec(workflow_desc['inputSpec'], skip_fields=get_advanced_inputs(workflow_desc, verbose)))
+                if workflow_desc.get('outputSpec') is not None and workflow_desc.get('outputs') is None:
+                    print_nofill_field("Output Spec", get_io_spec(workflow_desc['outputSpec']))
+                if  workflow_desc.get('inputs') is not None:
+                    print_nofill_field("Workflow Inputs", get_io_spec(workflow_desc['inputs']))
+                if  workflow_desc.get('outputs') is not None:
+                    print_nofill_field("Workflow Outputs", get_io_spec(workflow_desc['outputs']))
+                if 'stages' in workflow_desc:
```
Comment (geetduggal) [inline src/python/dxpy/utils/describe.py]: Minor, may want to add a note somewhere in the describe if there is an 'inputs' and 'outputs' field, smth like the following: 'Note: this workflow has an explicit input and output specification, and as such stage inputs cannot be modified at run-time.'

**Pull request merged**